Podfile

```diff
@@ -14,8 +14,8 @@ project 'AEPMessaging.xcodeproj'
 
 pod 'SwiftLint', '0.52.0'
 
-$dev_repo = 'https://github.com/adobe/aepsdk-core-ios.git'
-$dev_branch = 'dev-v5.3.0'
+$dev_repo = 'https://github.com/sbenedicadb/aepsdk-core-ios.git'
+$dev_branch = 'temp-utils'
```

Comment:
where are we using these branches
I see E2EFunctionalTests still used app_main definition and not app_dev
If so can we remove this